Tests/DPG_test.py

```
@@ -7,18 +7,22 @@
         return super().setUpClass()
 
     @classmethod
     def tearDownClass(cls):
-        dpg.destroy_context()
-        return super().tearDownClass()
\ No newline at end of file
+        for item in list(dpg.get_all_items()):
+            try:
+                dpg.delete_item(item)
+            except Exception:
+                pass
```

Ruff (0.15.17)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)